src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -49,3 +49,182 @@
 .tribe-theme-genesis .tribe-block__event-website a {
 	font-size: 1.65rem;
 }
+
+/* = Venue block view styles
+============================================= */
+.tribe-block__venue {
+	border-top: 1px solid #d5d5d5;
+	flex-direction: column-reverse;
+	justify-content: flex-start;
+	padding: 22.5px 0;
+
+	@media screen and (min-width: 768px) {
+		flex-direction: row;
+		padding: 32px 0;
+	}
+
+	&.tribe-block__venue--has-map {
+
+		.tribe-block__venue__meta,
+		.tribe-block__venue__map {
+			flex-basis: 100%;
+			width: 100%;
+
+			@media screen and (min-width: 768px) {
+				flex-basis: 49%;
+				width: 49%;
+			}
+		}
+	}
+
+	.tribe-block__venue__meta {
+		margin-top: 24px;
+
+		@media screen and (min-width: 768px) {
+			margin-top: 0;
+		}
+
+		.tribe-block__venue__name {
+
+			h3 {
+				font-size: 1rem;
+				font-weight: normal;
+				letter-spacing: normal;
+				line-height: 1.64;
+			}
+		}
+
+		.tribe-block__venue__address {
+			margin-bottom: 0;
+
+			a {
+				color: var(--tec-color-link-accent);
+				margin-top: 17px;
+			}
+
+			.tribe-region.tribe-events-abbr {
+				text-decoration: none;
+			}
+		}
+
+		.tribe-block__venue__phone {
+			margin-bottom: 0;
+		}
+
+		.tribe-block__venue__address,
+		.tribe-block__venue__phone,
+		.tribe-block__venue__website {
+			color: #141827;
+			font-weight: normal;
+			letter-spacing: normal;
+			line-height: 1.64;
+		}
+	}
+}
+
+/* Theme specific overrides */
+
+/* Twenty-twenty-one theme */
+.tribe-theme-twentytwentyone .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: var(--tec-color-link-accent);
+}
+
+/* Twenty-twenty theme */
+.tribe-theme-twentytwenty {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__address,
+			.tribe-block__venue__address a,
+			.tribe-block__venue__phone,
+			.tribe-block__venue__website {
+				font-size: 1.65rem;
+			}
+
+			.tribe-block__venue__website a {
+				color: var(--tec-color-link-accent);
+			}
+		}
+	}
+}
+
+/* Twenty Seventeen */
+.tribe-theme-twentyseventeen {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__website a {
+				color: var(--tec-color-link-accent);
+			}
+		}
+	}
+}
+
+/* Divi */
+.tribe-theme-divi {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
```

Comment:
no need to unnecessarily nest here

Reply:
I added nesting to this because down the line we need to combine the theme overrides for each block. You can see this in action on this [PR](https://github.com/the-events-calendar/the-events-calendar/pull/3611).